runelite-client/src/main/java/net/runelite/client/plugins/menuentryswapper/MenuEntrySwapperConfig.java

```diff
@@ -242,6 +242,16 @@ default boolean swapAbyssTeleport()
 		return true;
 	}
 
+	@ConfigItem(
+			keyName = "swapPlank",

```

Comment:
fix indent level

Reply:
To do that, do I need to cancel the pull request, make the fix, then resubmit?

Reply:
No, just made the changes, then commit and push.

Reply:
probably better to just fix the original commit with a reset & force push

Reply:
I think I've taken care of it now.